Makefile

```diff
@@ -3,26 +3,22 @@
 all: install client
 
 # Install all project dependencies
-install: install-node install-php
+install: install-pnpm install-node install-php
 
 # Install Node dependencies
 install-node:
-	npm install
+	pnpm install
+
+install-pnpm:
+	npm install -g pnpm@7.27.1
```

Comment:
This is the only thing I'd say is a strong blocker. I'd much rather have the extra line of instructions in our readme or something than having `make` implicitly change my global environment without any sort of warning!

Reply:
Mmm... fair enough

Reply:
It's on the development-environment.md, I'll remove it from the Makefile though